cicecore/cicedynB/general/ice_init.F90

```diff
@@ -2439,9 +2470,10 @@ subroutine set_state_var (nx_block, ny_block, &
                                 vicen,    vsnon, &
                                 uvel,     vvel)
 
+
       use ice_arrays_column, only: hin_max
       use ice_domain_size, only: nilyr, nslyr, nx_global, ny_global, ncat
-      use ice_grid, only: grid_type
+      use ice_grid, only: grid_type, grid_system, grid_average_X2Y
```

Comment:
Are grid_system, grid_average_X2Y used here, or can these be removed?

Reply:
These were not used and removed